packages/pwa-kit-react-sdk/src/ssr/server/react-rendering.js

```diff
@@ -75,9 +75,11 @@ const logAndFormatError = (err) => {
 // Because multi-value params are not supported in `aws-serverless-express` create a proper
 // search string using the `query` property. We pay special attention to the order the params
 // as best as we can.
-const getLocationSearch = (req) => {
+export const getLocationSearch = (req) => {
+    const {app} = getConfig()
```

Comment:
It would make testing a lot easier if you were to make the `interpretPlusSignAsSpace` config value an option in this function.

e.g.
```
export const getLocationSearch = (req, options) => {
   const {interpretPlusSignAsSpace = fasle} = opts
```

This way in your tests you don't have to do any extraneous mocking of the config function. It jsut so happens that we have access to the config in the function we are calling `getLocationSearch` from too.

Reply:
It is changed as suggested